metrotranscom/doorway
# Update seed data to bay area counties

# Pull Request Template
Since we removed the ability for Partners to utilize a county that is covered by other Bloom instances (SF, San Mateo, Alameda, and City of San Jose), then we need to update the seed data to counties outside of those areas so that when they are viewed in the partners portal the county drop down will show data. 

I tried to find addresses that were similar to the prior ones, in case it's easier to reference in the future. Not sure it really matters.

## Issue Overview

This PR addresses #issue


## How Can This Be Tested/Reviewed?
View main or any other branch with prior seed data & take a screenshot to compare with the new data. (You might need to add all counties to the filters to see all the data.)
![image](https://github.com/metrotranscom/doorway/assets/12400715/228ad750-0ad0-472c-9146-2a3f0cbbd1db)

Install & build all `backend/core` and `partners`. Reseed (`yarn db:reseed` from `backend/core`) data and then `yarn dev` all 3 services.

Public site should have the following listings with [doorway] in the title:
- 1 in Sausalito
- 1 in Fairfield
- 1 in Sonoma (Sesame St - broken previously & need to be fixed in subsequent PR)
- 1 in Santa Clara (Sesame St - broken previously & need to be fixed in subsequent PR)
- 13 in San Ramon

*you may need to adjust the filter to include all counties

then double check as many listings as you have time for on the Partners & Public site to ensure that the address data & maps still look valid and don't have any issues. The things that will change are the map pin location & the county label at the top of each listing
 ![image](https://github.com/metrotranscom/doorway/assets/12400715/c372806a-b1a2-4d84-bd52-e977344573e2)

Here's the locations updated in this PR:
![image](https://github.com/metrotranscom/doorway/assets/12400715/5a799098-529b-4233-9d11-2591dd3b99d1)

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
